src/shared/error/payload.ts

```diff
@@ -0,0 +1,24 @@
+import type { Show } from 'fp-ts/lib/Show'
+
+export const PayloadIsEmptyError = () =>
+  ({
+    _tag: 'payload-is-empty'
+  } as const)
+
+/**
+ * Error when a payload that is expected to contain data is empty.
+ */
+
+export type PayloadIsEmptyError = ReturnType<typeof PayloadIsEmptyError>
```

Comment:
Remove the empty line (between comment & code).

```suggestion
/**
 * Error when a payload that is expected to contain data is empty.
 */
export type PayloadIsEmptyError = ReturnType<typeof PayloadIsEmptyError>
```